converters/google/convert_test.go

```diff
@@ -47,3 +69,144 @@ func TestSortByName(t *testing.T) {
 		})
 	}
 }
+
+func TestAddResourceChanges_unknownResourceIgnored(t *testing.T) {
+	rc := tfplan.ResourceChange{
+		Address:      "whatever.google_unknown.foo",
+		Mode:         "managed",
+		Type:         "google_unknown",
+		Name:         "foo",
+		ProviderName: "google",
+		Change: tfplan.ChangeRepresentation{
+			Actions: []string{"change"},
+			Before:  nil,
+			After:   nil,
+		},
+	}
+	c, err := newTestConverter()
+	assert.Nil(t, err)
+	err = c.AddResourceChanges([]tfplan.ResourceChange{rc})
+	assert.Nil(t, err)
+	assert.EqualValues(t, map[string]Asset{}, c.assets)
+}
+
+func TestAddResourceChanges_unsupportedResourceIgnored(t *testing.T) {
+	rc := tfplan.ResourceChange{
+		Address:      "whatever.google_unknown.foo",
+		Mode:         "managed",
+		Type:         "google_unsupported",
+		Name:         "foo",
+		ProviderName: "google",
+		Change: tfplan.ChangeRepresentation{
+			Actions: []string{"change"},
+			Before:  nil,
+			After:   nil,
+		},
+	}
+	c, err := newTestConverter()
+	assert.Nil(t, err)
+
+	// fake that this resource is known to the provider; it will never be "supported" by the
```

Comment:
just curious, why is that?

Reply:
Since this is a fake terraform resource (`google_unsupported`) we can be confident that it is not going to have support added, so this test shouldn't accidentally break in the future. Does that answer the question?

Reply:
I see, I thought this was referring to `google_compute_disk`!